src/docs/taiko.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Taiko
```

Comment:
Imo, we should place  a "info callout" similar to the callout in the precompiles section where we state something of the sorts of:

"The rollup is still in development. Information presented here may change by the time Mainnet is launched."